src/com/owncloud/android/db/PreferenceManager.java

```diff
@@ -22,70 +22,120 @@
 import android.content.Context;
 import android.content.SharedPreferences;
 
+import com.owncloud.android.utils.FileStorageUtils;
+
 /**
  * Helper to simplify reading of Preferences all around the app
  */
-
-public class PreferenceManager {
-
+public abstract class PreferenceManager {
```

Comment:
because all it is a helper class with all methods being static so nobody should create instances but simply call the helper methods thus I made it abstract :)